pyproject.toml

```diff
@@ -44,10 +45,12 @@ dev-dependencies = [
 ]
 
 [tool.uv.sources]
+ethereum = { git = "https://github.com/kkrt-labs/execution-specs.git", branch = "kakarot-changes" }
```

Comment:
this should be in the `mpt` package no?

Reply:
ah, well i didn't make changes to ethereum, where is the ethereum dependency in main?

Reply:
probably a rebase conflict on your branch then